### ISSUE TITLE
Add YouTube and Social Media Download Capture

### ISSUE THREAD

**Issue by JARengine** [amir1376/ab-download-manager#283]
Please add download option in Extension or browser integrations that can able to download from Social media or Youtube platform, and this will be the best downloader trust me!

**Comment by push-on** [comment]
1st of all it's an open-source software. 2nd YouTube without explicit permission is generally against YouTube's Terms of Service and may infringe on copyright laws. YouTube's Terms of Service explicitly prohibit downloading content without authorization, except through features provided by YouTube itself. as an open-source software, it's hard to do so. for both YouTube and other social media and it's safe that way. I want to download from social media. but it will be risk. for a open-source tool like this

**Comment by MarkD1212** [comment]
Thanks for you great software. Is it possible to enable YT video download?

**Comment by 0-Satanael** [comment]
> 1st of all it's an open-source software. 2nd YouTube without explicit permission is generally against YouTube's Terms of Service and may infringe on copyright laws. YouTube's Terms of Service explicitly prohibit downloading content without authorization, except through features provided by YouTube itself. as an open-source software, it's hard to do so. for both YouTube and other social media and it's safe that way. I want to download from social media. but it will be risk. for a open-source tool like this

but how IDM allows download from youtube except websites with protected content?

**Comment by TecnikOfficial** [comment]
> but how IDM allows download from youtube except websites with protected content?

yes idm is goated tool , this ab download manager is also largely inspired from idm . due to idm multiple sites support i m not able to switch to any other download managers for the time being

**Comment by TecnikOfficial** [comment]
> 1st of all it's an open-source software. 2nd YouTube without explicit permission is generally against YouTube's Terms of Service and may infringe on copyright laws. YouTube's Terms of Service explicitly prohibit downloading content without authorization, except through features provided by YouTube itself. as an open-source software, it's hard to do so. for both YouTube and other social media and it's safe that way. I want to download from social media. but it will be risk. for a open-source tool like this

there are many i mean a lottt of github open source project even popular once that have youtube video download functionality so what u r saying is not correct
